packages/wallet/src/redux/protocols/direct-funding/readme.md

```diff
@@ -0,0 +1,25 @@
+# Direct Funding Protocol
+
+The direct funding protocol is one of the funding protocols, which means that the protocol progresses the channel from after the exchange of the PreFund setup states to where the participants are able to exchange PostFund states. The keeps track of adjudicator funding in order to track when it is safe to deposit funds.
+
+## State Machine
+
+The protocol implements the following state machine:
+
+TODO: add action names for transitions.
+
+```mermaid
+graph LR
+ST((Start))-->NSD(NotSafeToDeposit)
+NSD-->|FUNDING_RECEIVED|TSSM(TransactionSubmissionStateMachine)
```

Comment:
This should probably be the direct-funding state i.e. `WaitForDepositTransaction`.

Reply:
Addressed in https://github.com/magmo/apps/pull/388/commits/446b7d19cc2eb0467483c795955720bdd74241ac.